### PR TITLE
[WFLY-15024]:Move WildFly Preview to a native jakarta namespace varia…

### PR DESCRIPTION
**Pull request by jimma**
https://issues.redhat.com/browse/WFLY-15024

### CODE REVIEW

File: ee-9/pom.xml [hunk 1]
```diff
@@ -672,6 +673,17 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.jboss.spec.jakarta.xml.ws</groupId>
+                <artifactId>jboss-jakarta-xml-ws-api_3.0_spec</artifactId>
```
Comment (yersan) [inline ee-9/pom.xml:678]: I miss the license entry for this new artifact. The ee-8-api version has one for `org.jboss.spec.javax.xml.ws:jboss-jaxws-api_2.3_spec`. I would expect the same for the new one.

**Review by yersan — CHANGES_REQUESTED** [review]

File: ee-9/pom.xml (hunk 1 above)
Comment (jimma) [reply ee-9/pom.xml:678]: Added. Thanks for review.

File: ee-9/feature-pack/src/main/resources/license/preview-feature-pack-licenses.xml [hunk 2]
```diff
@@ -524,8 +524,8 @@
       </licenses>
     </dependency>
     <dependency>
-      <groupId>org.jboss.spec.javax.xml.ws</groupId>
-      <artifactId>jboss-jaxws-api_2.3_spec</artifactId>
+      <groupId>org.jboss.spec.jakarta.xml.ws</groupId>
+      <artifactId>jboss-jakarta-xml-ws-api_3.0_spec</artifactId>
```
Comment (bstansberry) [inline ee-9/feature-pack/src/main/resources/license/preview-feature-pack-licenses.xml:528]: Please move this up into the right alphabetical order spot based on the new GA.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Looks good except for the alphabetical order detail.

Please squash this once that's fixed and we can cross this one off the list!

File: ee-9/feature-pack/src/main/resources/license/preview-feature-pack-licenses.xml (hunk 2 above)
Comment (jimma) [reply ee-9/feature-pack/src/main/resources/license/preview-feature-pack-licenses.xml]: Fixed.

**Review by yersan — APPROVED** [review]
Jim, thanks. Changes look good to me. 

We don't have a Jira validating the uses of the JBoss forks instead of the eclipse ones for this spec, see https://issues.redhat.com/browse/WFLY-15098, so I assume that is intentional.

**Review by bstansberry — APPROVED** [review]
Looks good except for the alphabetical order detail.

Please squash this once that's fixed and we can cross this one off the list!

**Pull request merged**